### PR TITLE
created black list of names

### PR DESCRIPTION
**Pull request by i-am-jeetu**
A fix to issue #167 

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made.
- [x] I have added my changes to the `CHANGELOG.md`

### CODE REVIEW

**Comment by i-am-jeetu** [comment]
@sobolevn Can you see once why checks are failing? Locally I am passing all the tests.

File: dotenv_linter/violations/names.py [hunk 1]
```diff
@@ -134,3 +135,27 @@ class RawNameViolation(BaseFSTViolation):
...
+class ReservedNameViolation(BaseFSTViolation):
+    """
+    Restricts to use of blacklisted names.
+
+    Reasoning:
+        It does not make any sense to use such names.
+
+    Solution:
+        Change such namess.
+        Or, Add _ at the end, to distinguish it from reserved ones.
```
Comment (sobolevn) [inline dotenv_linter/violations/names.py:150]:
```
``_``
```

File: dotenv_linter/violations/names.py [hunk 2]
```diff
@@ -134,3 +135,27 @@ class RawNameViolation(BaseFSTViolation):
...
+
+    Example::
```
Comment (sobolevn) [inline dotenv_linter/violations/names.py:152]: Should be an empty line after `Example::`

File: dotenv_linter/violations/names.py [hunk 3]
```diff
@@ -134,3 +135,27 @@ class RawNameViolation(BaseFSTViolation):
...
+        # Wrong:
+        DJANGO_ENV=some_value
+
+    .. versionadded:: 0.1.0
```
Comment (sobolevn) [inline dotenv_linter/violations/names.py:156]: Please, specify a correct version.

**Comment by sobolevn** [comment]
Thanks a lot!

**Comment by sobolevn** [comment]
Great work 🎉

**Pull request merged**